testing/scripts/seldon_e2e_utils.py

```diff
@@ -4,43 +4,110 @@
 from seldon_core.proto import prediction_pb2_grpc
 import grpc
 import numpy as np
-from k8s_utils import *
+import time
+from subprocess import run, Popen
+import subprocess
+import json
+from retrying import retry
+import logging
 
+API_AMBASSADOR = "localhost:8003"
 
-def wait_for_rollout(deploymentName):
-    ret = run("kubectl rollout status -n test1 deploy/" + deploymentName, shell=True)
-    while ret.returncode > 0:
+
+def get_s2i_python_version():
+    completedProcess = Popen(
+        "cd ../../wrappers/s2i/python && grep 'IMAGE_VERSION=' Makefile | cut -d'=' -f2",
+        shell=True,
+        stdout=subprocess.PIPE,
+    )
+    output = completedProcess.stdout.readline()
+    version = output.decode("utf-8").rstrip()
+    return version
+
+
+def get_seldon_version():
+    completedProcess = Popen(
+        "cat ../../version.txt", shell=True, stdout=subprocess.PIPE
```

Comment:
Because `open("../../version.txt").read()` was too complicated... :joy:

But if it works, it works!

Reply:
Hahah if you're in the subprocess rabbithole might as well 😂 I'll update this one